testing/runner.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is used to create new test projects for the Google Cloud Run team
+set -e
+
+export TEAM_FOLDER=859004177471
```

Comment:
Should probably have TESTING_PROJECT in here, or possibly set CLOUDSDK_CORE_PROJECT environment variable instead of using the TESTING_PROJECT flag in the setup script. We can sort that out in follow-up in interest of closing this if you prefer.

Reply:
TESTING_PROJECT is set in the setup script:

`export TESTING_PROJECT=cloud-run-samples-test-${PROJECT_SUFFIX}`

I did add a `gcloud config set project cloud-run-samples-test` so it would pull the correct billing account.

Reply:
The purpose of this script is to pull all the details specific to our environment out of the setup script. The script is meant to operate as a shell-based terraform to spin up new project instances for our use or anyone wanting to test samples in the same environment. It also facilitates creating other scripts: such as running the deploy script of a Cloud Function so it deploys to the same project.

Reply:
`gcloud config set project` will allow the deploy_function.sh and other scripts to work.  However, deploy_function does assume that it's running in that function directory.

Reply:
We can revise that as we go. If a script is going to do `gcloud config set project` it should revert to the original project after it's done. Probably via an exit trap. We can add that as a follow-up if you want to get this merged :)